eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -489,6 +489,24 @@
           "old": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "new": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "justification": "Removing Jackson annotations from Job Router in transition to stream-style."
+        },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.(attributeRemoved|attributeAdded)",
+          "old": "class com\\.azure\\.resourcemanager\\.cosmos\\.models.*",
+          "justification": "Jackson annotation changed."
+        },
+        {
+          "regex": true,
+          "code": "java\\.method\\.removed",
+          "old": "method java\\.lang\\.Integer",
```

Comment:
I don't quite get this line. Why is `java.lang.Integer` been affected?

Reply:
This has been rollbacked.